pb_plugins/dcsdkgen/methods.py

```diff
@@ -141,10 +175,11 @@ def __init__(
             requests,
             responses):
         super().__init__(plugin_name, package, pb_method, requests, responses)
-        self._name = decapitalize(pb_method.name) + "Observable"
-        self._capitalized_name = pb_method.name + "Observable"
+        self._name = remove_subscribe(decapitalize(pb_method.name))
```

Comment:
That changes the behavior, right? It means that the Swift template now needs to add "Observable". Is that correct?

Reply:
Yes, it makes the name more generalized. The SWIFT templates have to be adjusted in some places, I'll do that asap!